microprofile/WFLY-10712_microprofile_metrics.asciidoc

```diff
@@ -267,8 +281,10 @@ For example the transaction metrics will be retrieved from the http://wildscribe
 * number-of-aborted-transactions
 * etc.
 
-When the `micrprofile-smallrye-metrics` is installed, it will browse WildFly Resource Model and find every metrics registered by resources from the model.
+When the `micrprofile-smallrye-metrics` is installed, it will browse WildFly Resource Model and find every metrics registered by subsystem resources from the model.
+If will only expose metrics from the subsystems specified by the `exposed-subsystems` attribute (or all if the wildcard `*` is used).
```

Comment:
Probably typo: If -> It

Reply:
How will this be utilized in our OpenShift images?  Will this form of configurability allow us to expose the desired metrics without exposing way more than is desired?

This style may be ok. Looking at the metrics declared in the hawkular agent config that this subsystem will replace, I don't see JGroups metrics, which AIUI are the problematic ones. There are some runtime attributes mentioned from JGroups, but not metrics.

https://github.com/jboss-openshift/cct_module/blob/master/os-eap70-hawkular/added/hawkular-javaagent-config.yaml